app/views/feedback.py

```diff
@@ -100,7 +99,6 @@ def post_thank_you():
     return redirect(url_for('feedback.thank_you'))
 
 
-@template_helper.with_session_timeout
-@template_helper.with_analytics
-def _render_template(template, **kwargs):
-    return template_helper.render_template(template, **kwargs)
+@with_session_timeout
+def _render_template_with_session_timeout(template, **kwargs):
```

Comment:
I think we should remove the `with_session_timeout` decorator and either have an explicit `render_template_with_session_timeout` method in `template_helper.py` or add it as an argument to `render_template` in `template_helper.py`.

Reply:
Didn't bother with a helper method in the end. Have just passed it in `quesstionnaire.py` and `feedback.py`.